GitExtUtils/GitUI/ThreadHelper.cs

```diff
@@ -113,6 +113,11 @@ public static void FileAndForget(this Task task, Func<Exception, bool> fileOnlyI
                 });
         }
 
+        public static async Task JoinPendingOperationsAsync()
```

Comment:
📝 This method was intentionally changed to require a `CancellationToken`. This method is not allowed to be called from production code, and we need the cancellation support in tests to avoid hangs. There is no valid call possible to this method, so it should be removed.

Reply:
Look at the logs and read the 'try' in the commit message. The failed tests haven't wait for completion. They have been cancelled at once.

Reply:
removed